apps/shared/fonts.css

```diff
@@ -0,0 +1,45 @@
+@font-face {
```

Comment:
I'd like this to be in a lib, like `libs/island-ui/fonts`.

Can that be referenced in a styles array in workspaces.json, or from css like this:

```
@import '@island.is/island-ui/fonts';
```

I guess not, since these path aliases are defined in tsconfig rather than webpack, but worth trying.

Reply:
I've moved this to libs but was unable to import like you mention 😕